custom-vu/src/main/kotlin/jces1209/vu/page/project/CloudProjectNavigatorPage.kt

```diff
@@ -0,0 +1,25 @@
+package jces1209.vu.page.project
+
+import jces1209.vu.page.CloudIssueNavigator
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+
+class CloudProjectNavigatorPage(
+    private val driver: WebDriver
+) : ProjectNavigatorPage {
+
+    override fun openProject(projectKey: Int) {
+        driver
+            .wait(ExpectedConditions
+                .elementToBeClickable(By.xpath("(//*[@data-testid='Content']//a)[$projectKey]")))
```

Comment:
`[@data-testid='Content']` is the total page container
use this one `data-test-id="global-pages.directories.directory-base.content.table.container"`